### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by manotejmeka** [manotejmeka/MI-449-SS17-741-css-stealing-styles-xedvDK#1]
@stuartpearman Can you take a look at this? It's [hosted here](https://manotejmeka.github.io/MI-449-SS17-741-css-stealing-styles-xedvDK/) and meets the following criteria:

- [x] The page includes the name of your search engine in big letters (you can make it up), with a search `input` below it, then a button below that to run the search (the button doesn't have to do anything for this prototype)
- [x] The font size of the search engine name is at least 70 pixels, so that it's very easy to see
- [x] `span` elements are wrapped around each letter of the name, like this: `<span>G</span><span>o</span><span>o</span><span>g</span><span>l</span><span>e</span>`, so that they can be styled individually with classes
- [x] The on-hover colors from the paragraphs [on this page](http://skillful-clouds.surge.sh/) are used to color individual letters of your name (similar to the letters in Google's logo)
- [x] Inspect the `input` element at the bottom of the [_Prototyping states_ page of the lesson](https://msu.lansing.codes/courses/MI-000-SS00-000/lessons/css-stealing-styles/5) to create an identical-looking search box for your own site (it should even use the same styles on `focus`)
- [x] The styles from the "Create Account" button [on this page](http://plant-servant.surge.sh/) are used for your "Search" button (it should even use the same styles on `hover` and `active`)
- [x] There are no stylelint errors in your CSS (this will also be expected for all future projects)
- [x] All your CSS exists in a `.css` file

**Comment by stuartpearman** [comment]
This is sweet Manotej, you're a :star:

I've reworked this to use the CSS `hover` pseudo class in javascript, just in case you're curious:

[Code](https://gist.github.com/stuartpearman/aa9ab975b33d9e743dba8b9540acf372)

[Live example](http://codepen.io/interknights/pen/pROOaB)

You don't have to change yours, just wanted to show you another weird thing that's possible with javascript.

The only criteria that you're missing is #2:

> The font size of the search engine name is at least 70 pixels, so that it's very easy to see

Yours works out to be 50px, you can bump it up a lot bigger so it's easy to see the hover effect.

-Stu

**Comment by manotejmeka** [comment]
@stuartpearman Thank you for the example code you provided with the hover effect. 
I pushed my code with the font element fixed.
I am confused a little with you code. "css = '.title-' + i + ':hover { color: ' + CSS_COLOR_NAMES[i] + '; } '" For the line of code is the css a variable that holds text? So are you just creating a new css style sheet and writing into the style sheet to get the hover effect?

**Comment by stuartpearman** [comment]
Yes, so first I create a new `<style></style>` element with this line:

    var style = document.createElement('style');

Then, looping through each letter, I add a class to each span. `.title-1` ,`.title-2`, etc.

I also create a variable `css` where I store the css for the span. It'll look something like this:

    .title-1:hover { color: DarkBlue }

If the `style` tag already has some css associated with it, I add the new css to the end, otherwise I use the DOM property `style.stylesheet.cssText` to create some new CSS.

Once the loop is done, I append the `style` tag to the `head` with all the CSS inside.

Does that make sense?

**Comment by manotejmeka** [comment]
@stuartpearman  Yes that makes sense. Thank you

**Comment by stuartpearman** [comment]
Hey Manotej, I forgot that I actually requested change for the font-size. From above comment:

The only criteria that you're missing is #2:

> The font size of the search engine name is at least 70 pixels, so that it's very easy to see

Yours works out to be 50px, you can bump it up a lot bigger so it's easy to see the hover effect.

**Comment by manotejmeka** [comment]
@stuartpearman Sorry I did fix it, but forgot to push it to you. It is pushed now

**Comment by stuartpearman** [comment]
Looks good! :shipit:
